docs/fiddles/features/macos-dock-menu/index.html

```diff
@@ -7,10 +7,5 @@
 </head>
 <body>
     <h1>Hello World!</h1>
-    <p>
```

Comment:
Maybe we can leave a small description in the `<p>` tag to describe what action the user can take (right-clicking the dock icon to see the app dock menu).

Reply:
I like it! I thought about it, but since it wasn't in the original, wasn't sure. I'll add this for all my current PR's